include/WOCStdLib/winnt.h

```diff
@@ -0,0 +1,23 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#pragma once
+
+#include_next <winnt.h>
+#ifdef __OBJC__
+#undef DELETE
```

Comment:
the "new hotness" for this is to use a pragma push which will push the old def onto to macro stack. You can then undef / redef things to your heart's content and pop at the end. This a convenient way to make sure the window's header gets what it wants but doesn't corrupt state down the line without the big hammer of killing anyone else's DELETE macro ... 

---
In reply to: [74333570](https://github.com/Microsoft/WinObjC/pull/866#discussion_r74333570) [](ancestors = 74333570,74333399)